server/apiCalls/transformers/analyses.js

```diff
@@ -11,8 +11,9 @@ const { findAnalysesForOfferingId } = require('./docs')
 const _analysesPerCourseOffering = async (parsedOfferings, analyses) => {
   const courseOfferings = []
   await parsedOfferings.forEach(offering => {
-    const { courseCode, firstSemester } = offering
-    const offeringId = Number(offering.offeringId)
+    const { courseCode, firstSemester, courseRoundApplications } = offering
+    const [courseRoundApplication] = courseRoundApplications
+    const { course_round_application_code: offeringId } = courseRoundApplication
```

Comment:
according to the old code it has to be Number(offeringId), must be checked why

Reply:
Removed